pages/common/svgo.md

```diff
@@ -0,0 +1,56 @@
+# svgo
+
+> SVG Optimizer: a Nodejs-based tool for optimizing SVG vector graphics files.
+> It applies a series of transformation rules (plugins), which can be toggled individually.
+
+- Optimize a file using the default plugins (overwrites the original file):
+
+`svgo {{test.svg}}`
+
+- Optimize a file and save the result to another file:
+
+`svgo {{test.svg}} {{test.min.svg}}`
+
+- Optimize all SVG files within a folder (overwrites the original files):
+
+`svgo -f {{path/to/folder/with/svg/files}}`
+
+- Optimize all SVG files within a folder and save the resulting files to another folder:
+
+`svgo -f {{path/to/input/folder}} -o {{path/to/output/folder}}`
+
+- Optimize SVG content passed from another command, and save the result to a file:
+
+`{{cat test.svg}} | svgo -i - -o {{test.min.svg}}`
+
+- Optimize a file and print out the result:
+
+`svgo {{test.svg}} -o -`
+
+- Optimize SVG content passed as a string, and save the result to a file:
```

Comment:
IMO, the example starting from this one, and the following 2 can be left out. We have already covered passing content from another command above. And the base64 data URI and compressing the resulting file seems a bit too much.